pkg/actions/diagnostics.go

```diff
@@ -339,13 +339,11 @@ func gatherCodewindEclipseLogs(codewindEclipseWSDir string) {
 		} else {
 			warnDG("Unable to collect Eclipse logs", "workspace metadata directory not found")
 		}
-	} else {
-		warnDG("Unable to collect Eclipse logs", "workspace not specified")
 	}
 }
 
 func gatherCodewindVSCodeLogs() {
-	logDG("Collecting VSCode logs ... ")
+	logDG("Attempting to collecting VSCode logs from default location ... ")
```

Comment:
If you moved this after the switch block you could say what the default location was.

Reply:
OK, done